File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.js
/**
 * @param {number[]} nums
 * @param {number} original
 * @return {number}
 */
var findFinalValue = function(nums, original) {
    return mul(nums,original);
};
function mul(nums,original)
{
    for(let i=0;i<nums.length;i++)
    {
        if(nums[i]==original)
        {
            return mul(nums,original*2);
        }
    }
    return original;
}

File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.java
class Solution {
    public int findFinalValue(int[] nums, int original) {
      return find(nums,original);
    }
    public static int find(int[] nums,int original)
    {
       for(int i=0;i<nums.length;i++)
       {
        if(nums[i]==original)
        {
            return find(nums,original*2);
        }

       }
       return original;
    }
}